app/client/lib/helpers/helperFunctions.js

```diff
@@ -191,6 +191,10 @@ Helpers.showNotification = function(i18nText, values, callback) {
   if (typeof mist !== 'undefined') mist.sounds.bip();
 };
 
+var multipleCaseAddresses = function(address) {
```

Comment:
generates both cases: checksum and lowercase. those are used in queries below.

@marcgarreau should you encounter any other issue (i couldn't with the limited time for testing), i can bulletproof it even more.

Reply:
👆 I performed the aforementioned changes.